### PR TITLE
drop get_state

### PR DESCRIPTION
**Pull request by 0lvin**

### CODE REVIEW

File: server_plugin/server.py [hunk 1]
```diff
@@ -189,6 +195,9 @@ def delete(vca_client, **kwargs):
 @operation
 @with_vca_client
 def get_state(vca_client, **kwargs):
```
Comment (achirko) [inline server_plugin/server.py]: You also need to remove get_state(). If we leave it, _get_state() will be processed twice - in start() and in get_state()

File: server_plugin/server.py [hunk 2]
```diff
@@ -156,6 +157,11 @@ def start(vca_client, **kwargs):
             raise cfy_exc.NonRecoverableError("Could not create vApp")
         wait_for_task(vca_client, task)
 
+    if not _get_state(vca_client, ctx):
+        return ctx.operation.retry(
+            message='Waiting for status',
```
Comment (achirko) [inline server_plugin/server.py:162]: Message could be more descriptive, something like "Waiting for VM's configuration to complete"

File: server_plugin/server.py (hunk 1 above)
Comment (0lvin) [inline server_plugin/server.py]: If drop this function i have error: 
executing bootstrap validation
mapping error: server_plugin.server has no attribute 'get_state' [node=manager_server, type=operations]

**Comment by achirko** [comment]
Looks like too many commits. One commit should be enough.

**Pull request closed without merging**